### PR TITLE
chore: project addition added Unit Converter

### PR DESCRIPTION
**Pull request by PatilHarshh**
Unit converter Added

## Related Issue

Issue no #1241

### CODE REVIEW

**Comment by PatilHarshh** [comment]
@priyankarpal Please Check

File: src/DB/projects.json [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+        "YouTube": "YOUR_YOUTUBE_ACCOUNT_LINK",
+        "Twitter": "https://twitter.com/Harsh94569157"
+      },
+      "Projects": [
+        {
+          "link": "https://github.com/PatilHarshh/Unit-Converter",
+          "title": "Unit-Converter",
+          "description": "This is the Unit Converter Project Basic Logic of Javascript and Begineer Friendly Project",
+          "tech": ["Javascript", "HTML","CSS"]
+        }
```
Comment (priyankarpal) [inline src/DB/projects.json:17]: you already added your project before so just paste the code to the previous one. check the contributing guide

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json (hunk 1 above)
Comment (PatilHarshh) [reply src/DB/projects.json:17]: ok

**Comment by PatilHarshh** [comment]
@priyankarpal can you see Now

**Review by PatilHarshh — COMMENTED** [review]
done

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**